src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsBuildWrapper.java

```diff
@@ -112,7 +112,31 @@ public OutputStream decorateLogger(AbstractBuild build, OutputStream logger) {
             }
         }
 
-        return new MaskPasswordsOutputStream(logger, allPasswords);
+        return new FilterImpl(allPasswords);
+    }
+
+    private static final class FilterImpl extends ConsoleLogFilter implements Serializable {
+
+        private static final long serialVersionUID = 1;

```

Comment:
Yes I know about the syntax for `long` literals, which is not necessary when you are assigning to a `long`-typed field.